x/stake/inflation.go

```diff
@@ -5,8 +5,8 @@ import (
 )
 
 const (
-	hrsPerYr  = 8766 // as defined by a julian year of 365.25 days
-	precision = 1000000000
+	hrsPerYr  = 8766         // as defined by a julian year of 365.25 days
+	precision = 100000000000 // increased to this precision for accuracy with tests on tick_test.go
```

Comment:
as stated on the previous closed PR, i increased precision, because for very large provision handouts the precision was cutting off 1 token, in rare cases. However this is dependant on the amount of tokens we test with. if we picked 10000000000000000000 tokens, provisions would be huge, and would need even more precision. however we are good with this amount of provision for now